From a22c6d90c0d7cecbde923014b398787fe56978c1 Mon Sep 17 00:00:00 2001
Message-Id: <a22c6d90c0d7cecbde923014b398787fe56978c1.1536572113.git.tredaelli@redhat.com>
In-Reply-To: <b60115a1c00e30e15395ca0c4bcaf22c3ac431cb.1536572113.git.tredaelli@redhat.com>
References: <b60115a1c00e30e15395ca0c4bcaf22c3ac431cb.1536572113.git.tredaelli@redhat.com>
From: Timothy Redaelli <tredaelli@redhat.com>
Date: Tue, 31 Jul 2018 15:15:28 +0200
Subject: [PATCH 2/2] net/mlx5: avoid stripping the glue library

Stripping binaries at build time is usually a bad thing since it makes
impossible to generate (split) debug symbols and this can lead to a more
difficult debugging.

Fixes: 59b91bec12c6 ("net/mlx5: spawn rdma-core dependency plug-in")
Cc: stable@dpdk.org

Signed-off-by: Timothy Redaelli <tredaelli@redhat.com>
Acked-by: Luca Boccassi <bluca@debian.org>
Acked-by: Christian Ehrhardt <christian.ehrhardt@canonical.com>
(cherry picked from commit c7684b6be4977e7d343b17f798192062b312461d)
---
 drivers/net/mlx5/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/mlx5/Makefile b/drivers/net/mlx5/Makefile
index 589a06798..d4cba7329 100644
--- a/drivers/net/mlx5/Makefile
+++ b/drivers/net/mlx5/Makefile
@@ -184,7 +184,7 @@ $(LIB): $(LIB_GLUE)
 $(LIB_GLUE): mlx5_glue.o
 	$Q $(LD) $(LDFLAGS) $(EXTRA_LDFLAGS) \
 		-Wl,-h,$(LIB_GLUE) \
-		-s -shared -o $@ $< -libverbs -lmlx5
+		-shared -o $@ $< -libverbs -lmlx5
 
 mlx5_glue.o: mlx5_autoconf.h
 
-- 
2.17.1

